.gitignore

```diff
@@ -24,6 +24,32 @@ manifest.mf
 npm-debug.log
 core/src/main/web/app/templates.js
 core/config/hash
-core/src/main/web/vendor/
 core/src/main/web/index.html
 /bin/
+
+core/src/main/web/vendor/bower_components/jquery/test
```

Comment:
core/src/main/web/vendor/bower_components/

shouldn't just that one take care of all of them?